introduction.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html>
+<head>
+ <meta charset="UTF-8">
+ <title>My introduction</title>
+</head>
+<body>
+ <h1>Eunbae Jeon</h1>
+ <p>Hi, my name is Eunbae Jeon. I'm a student at Fastcampus studying front-end engineering.</p>
```

Comment:
could you divide this paragraph into 2 paragraphs?

Reply:
Got it.